src/standalone/Soil/soil_hydrology_parameterizations.jl

```diff
@@ -392,3 +396,101 @@ function viscosity_factor(T::FT, γ::FT, γT_ref::FT) where {FT}
     Theta = FT(exp(factor))
     return Theta
 end
+
+"""
+    ice_fraction(θ_l::FT, θ_i::FT, ν::FT, θ_r::FT, ρ_l::FT, ρ_i::FT)::FT 
+where {FT}
+
+Computes and returns the
+fraction of the humidity in the pore space 
+that is due to sublimation.
+
+f = θ_iρ_i/(θ_iρ_i+(θ_l-θ_r)ρ_l)
+"""
```

Comment:
"""
    ice_fraction(θ_l::FT, θ_i::FT, ν::FT, θ_r::FT, ρ_l::FT, ρ_i::FT)::FT where {FT}

Return the fraction of the humidity in the pore space that is due to sublimation.

```math
f = \\frac{θ_i ρ_i}{θ_i ρ_i + (θ_l - θ_r) ρ_l}
````
"""

Reply:
This doesn't render well on github, but you can add a math block so that it renders as equation int he docs

Reply:
\```math
f = \\frac{θ_i ρ_i}{θ_i ρ_i + (θ_l - θ_r) ρ_l}
\```